### DISCUSSION TITLE
Is there a problem with Youtube video downloads?

### DISCUSSION THREAD

**Discussion by suatcamillo** [aclap-dev/video-downloadhelper#495] [Help and Q&A] [answered]
### Your question

When I try to download a youtube video it looks like it starts but turns red next instance and gives an error that says "Short content 1555".

Is this the result of YouTube's latest war against third-party apps?

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 0b1a3631d17e4f27
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "vdhcoapp",
  "version": "1.6.3",
  "binary": "C:\\Program Files\\net.downloadhelper.coapp\\bin\\net.downloadhelper.coapp-win-64.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "home": "C:\\Users\\sevencan",
  "converterBinary": "C:\\Program Files\\net.downloadhelper.coapp\\converter\\build\\win\\64\\ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "4.1.5"
}

**Comment by jcfrog** [comment, marked as answer]
Hi,
1. could you update your coapp? https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-Installation
2. I can't reproduce this issue, can you share a link with this problem?

**Reply by suatcamillo** [reply]
Thanks a lot! That solved the issue.
